README.md

```diff
@@ -35,15 +35,31 @@ your needs. The format is shown in the example. The simplest form would be:
 ```
 
 *Important Note:* You have to restart foreman in order to apply changes in
-`default_hostgroup.yaml`!
+`foreman_default_hostgroup.yaml`!
```

Comment:
@GregSutcliffe is this correct?  The rpm installs this as foreman_default_hostgroup.yaml but that must be just the rpm.  So I am not sure what to do here.  I can update the gem spec if needed and change the filename.